pandas/_libs/tslibs/strptime.pxd

```diff
@@ -14,6 +14,8 @@ cdef class DatetimeParseState:
     cdef:
         bint found_tz
         bint found_naive
+        bint found_naive_str
```

Comment:
Is it easy to add comments inline with these members or as part of the class? I can see the distinction between `found_native` and `found_naive_str` being unclear over time

Reply:
will update with comment(s).  we'll be able to simplify this a ton in 3.0 once deprecation are enforced